modules/admin-ui/src/main/webapp/scripts/shared/partials/modals/edit-events-modal.html

```diff
@@ -1,6 +1,6 @@
 <section ng-form="editEventsForm" ng-show="open" ng-keyup="keyUp($event)" tabindex="1" class="modal wizard modal-animation ng-hide" ng-controller="EditEventsCtrl">
     <header>
-        <a class="fa fa-times close-modal" ng-click="close()"></a>
+        <a class="fa fa-times close-modal" ng-click="clearAndClose()"></a>
```

Comment:
This would not fic the problem if you hit escape to close the modal, would it?

Reply:
Oh, you might be right, good catch!